frontend/wasm/src/editor_api.rs

```diff
@@ -235,10 +235,17 @@ impl JsEditorHandle {
 			let f = std::rc::Rc::new(RefCell::new(None));
 			let g = f.clone();
 
-			*g.borrow_mut() = Some(Closure::new(move || {
+			*g.borrow_mut() = Some(Closure::new(move |timestamp| {
 				wasm_bindgen_futures::spawn_local(poll_node_graph_evaluation());
 
 				call_closure_with_editor_and_handle(|editor, handle| {
+					let micros: f64 = timestamp * 1000.0;
```

Comment:
A super tiny nit pick: we standardize on the style of `1000.` instead of `1000.0`, which unfortunately rustfmt doesn't let us enforce automatically. Feel free to change this if your inclined, although it's not a big deal.